epochStart/metachain/epochStartData.go

```diff
@@ -106,21 +107,21 @@ func (e *epochStartData) VerifyEpochStartDataForMetablock(metaBlock *block.MetaB
 	}
 
 	if !bytes.Equal(receivedEpochStartHash, createdEpochStartHash) {
-		log.Warn("RECEIVED epoch start data")
-		displayEpochStartData(&metaBlock.EpochStart)
-
-		log.Warn("CREATED epoch start data")
-		displayEpochStartData(startData)
+		displayEpochStartData("received", receivedEpochStartHash, &metaBlock.EpochStart)
+		displayEpochStartData("created", createdEpochStartHash, startData)
 
 		return process.ErrEpochStartDataDoesNotMatch
 	}
 
 	return nil
 }
 
-func displayEpochStartData(startData *block.EpochStart) {
+func displayEpochStartData(mode string, hash []byte, startData *block.EpochStart) {
+	log.Warn(mode+" epoch start data",
+		"hash", hash, "startData", spew.Sdump(startData))
+
 	for _, shardData := range startData.LastFinalizedHeaders {
-		log.Debug("epoch start shard data",
+		log.Warn("epoch start shard data",
```

Comment:
change back to DEBUG ?

Reply:
no, as the message that indicate a serious problem is on WARN level, the details are on debug level